### DISCUSSION TITLE
Marry Monica and Mindmap

### DISCUSSION THREAD

**Discussion by niladrishekhar** [monicahq/monica#6453] [Ideas]
Is it possible to marry Monica and Mindmap (any tool - not a specific one) ?

Mostly like a visual representation of all my contacts in one screen, where I get to see who all is there - whats the relationship between them for example group of certain tags give a list of names related to the tag - check what actions are upcoming - and plan around them.

**Comment by Szeraax** [comment]
You would probably be best off trying to export the data as json and import into some graph visualizer.
